### DISCUSSION TITLE
How to get notification not only with indexes, but with object keys also?

### DISCUSSION THREAD

**Discussion by ukushu** [realm/realm-cocoa#8609] [Q&A]
What is use cases for indexes that is not relevant?

If was delted item on idx 10, but modified item on idx 12 and isertion on idx 15 this information already not useful, because of you cannot be sure what objects/keys was deleted and what was updated.

How can I recieve notifications not only with indexes, but also with keys?

**Comment by nirinchev** [comment]
You can't right now. The indices are useful if you're displaying a collection in a UITableView/CollectionView and need to animate changes.
